platform-core-module/src/main/java/com/softicar/platform/core/module/CoreCssClasses.java

```diff
@@ -15,17 +15,17 @@ public interface CoreCssClasses {
 	ICssClass PAGE_HEADER_DIV = new CssClass("PageHeaderDiv", CoreCssFiles.PAGE_STYLE);
 	ICssClass PAGE_HEADER_PATH_DIV = new CssClass("PageHeaderPathDiv", CoreCssFiles.PAGE_STYLE);
 	ICssClass PAGE_HEADER_PAGE_NAME = new CssClass("PageHeaderPageName", CoreCssFiles.PAGE_STYLE);
+	ICssClass PAGE_HEADER_NAVIGATION_TOGGLE_BUTTON = new CssClass("PageHeaderNavigationToggleButton", CoreCssFiles.PAGE_STYLE);
 	ICssClass PAGE_HEADER_SEPARATOR = new CssClass("PageHeaderSeparator", CoreCssFiles.PAGE_STYLE);
-	ICssClass PAGE_HEADER_USER_DISPLAY = new CssClass("PageHeaderUserDisplay", CoreCssFiles.PAGE_STYLE);
 	ICssClass PAGE_NAVIGATION_COLLAPSED = new CssClass("PageNavigationCollapsed", CoreCssFiles.PAGE_NAVIGATION_STYLE);
 	ICssClass PAGE_NAVIGATION_DIV = new CssClass("PageNavigationDiv", CoreCssFiles.PAGE_NAVIGATION_STYLE);
 	ICssClass PAGE_NAVIGATION_FOLDER_DIV = new CssClass("PageNavigationFolderDiv", CoreCssFiles.PAGE_NAVIGATION_STYLE);
 	ICssClass PAGE_NAVIGATION_FOLDER_CHEVRON = new CssClass("PageNavigationFolderChevron", CoreCssFiles.PAGE_NAVIGATION_STYLE);
 	ICssClass PAGE_NAVIGATION_FOLDER_CONTENT_DIV = new CssClass("PageNavigationFolderContentDiv", CoreCssFiles.PAGE_NAVIGATION_STYLE);
 	ICssClass PAGE_NAVIGATION_FOLDER_TITLE_DIV = new CssClass("PageNavigationFolderTitleDiv", CoreCssFiles.PAGE_NAVIGATION_STYLE);
+	ICssClass PAGE_NAVIGATION_LINK_DIV = new CssClass("PageNavigationLinkDiv", CoreCssFiles.PAGE_NAVIGATION_STYLE);
+	ICssClass PAGE_NAVIGATION_LINK_TITLE = new CssClass("PageNavigationLinkTitle", CoreCssFiles.PAGE_NAVIGATION_STYLE);
```

Comment:
now that you wiped the term `link`, shouldn't this be like this
```suggestion
	ICssClass PAGE_NAVIGATION_ENTRY_DIV = new CssClass("PageNavigationEntryDiv", CoreCssFiles.PAGE_NAVIGATION_STYLE);
	ICssClass PAGE_NAVIGATION_ENTRY_TITLE = new CssClass("PageNavigationEntryTitle", CoreCssFiles.PAGE_NAVIGATION_STYLE);
```